tembo-operator/src/app_service/ingress.rs

```diff
@@ -269,7 +269,7 @@ pub fn generate_ingress_tcp_routes(
                             continue;
                         }
 
-                        let matcher = format!("{host_matcher} && PathPrefix(`{}`)", path);
+                        let matcher = format!("{}", host_matcher_tcp);
```

Comment:
Can we just pass `host_matcher_tcp` directly to the IngressRouteTCPRoutes, instead of re-assigning it here. There's no changes being made to it, right?

Something similar to this...

```rust
                        let route = IngressRouteTCPRoutes {
                            r#match: host_matcher_tcp,
                            services: Some(vec![IngressRouteTCPRoutesServices {
                                name: resource_name.to_string(),
```